### DISCUSSION TITLE
Switch to workspace with 1 key

### DISCUSSION THREAD

**Discussion by ascii78** [WayfireWM/wayfire#1437] [General]
I'd like to switch to a certain workspace with a function key. I've tried:

```
[vswitch]
binding_1 = KEY_F1
binding_2 = KEY_F2
```
In sway I use the following:

```
bindsym F1 workspace $ws1
bindsym F2 workspace $ws2
```
Is this possible ?

**Comment by ammen99** [comment]
Yes, wayfire-git version supports this. See https://github.com/WayfireWM/wayfire/wiki/Configuration#vswitchbinding_n

**Reply by ascii78** [reply]
Thanks, it works, I was not on the -git version
